openassessment/templates/openassessmentblock/message/oa_message_closed.html

```diff
@@ -1,21 +1,15 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% if not_yet_open %}
-            {% trans "This Task Is Not Yet Available" %}
-        {% else %}
-            {% trans "This Assignment Has Closed" %}
-        {% endif %}
-    </h3>
+    <h5 class="message__title">{% trans "Status" %}</h5>
     <div class="message__content">
         <p>
             {% if not_yet_open %}
-                {% trans "Check back to complete the assignment once this section has opened." %}
+                {% trans "This Task Is Not Yet Available. Check back to complete the assignment once this section has opened." %}
             {% else %}
-                {% trans "One or more deadlines for this assignment have passed. You will receive an incomplete grade for this assignment." %}
+                {% trans "This Assignment Has Closed. One or more deadlines for this assignment have passed. You will receive an incomplete grade for this assignment." %}
```

Comment:
The titles that have been moved into the message body should be sentence case: "This assignment has closed."